scripts/translation/lib/RevcheckFileList.php

```diff
@@ -4,26 +4,30 @@
 
 class RevcheckFileList
 {
-    private $list;
+    var $list = array();
```

Comment:
Why use ``var`` instead of ``public``?

Reply:
No typed arrays in PHP, and access trough `get()` to document the possible values.

Algo, the `$list` is constructed from file system, and is not expected of itens to be added or removed trom this list after the inicial loading.

Reply:
Why was it made public at all? You can still add the `array` type and then add proper type in docblock. Somethign like `string[]`. 

But I agree we should not use the old alias of `public` anymore.

Reply:
Replaced var.